pkg/asset/cluster/tfvars.go

```diff
@@ -390,6 +390,7 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			clusterID.InfraID,
 			caCert,
 			bootstrapIgn,
+			installConfig.Config.ControlPlane.Platform.OpenStack,
```

Comment:
You dont have to do this in this patch, but could we just pass the install config here, or make a new struct to pass tfvars? The number of arguments to this function is becoming ludicrous :)

Reply:
I know... Abhinav asked the same question https://github.com/openshift/installer/pull/3291#discussion_r403138644 
But I think we don't have other options, because we need to differenciate between primary and additional objects, and the only way is to read them from the machine pool :(